packages/amplify-cli/src/__tests__/extensions/amplify-helpers/delete-project.test.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
how commonly do we disable those linter rules (and the other one below) in tests? If it's a pattern perhaps tests could use different set of rules.

Reply:
very often, I'm not sure why yet, but tests have a unique import resolution strategy that I don't understand